### PR TITLE
Added bootstrap tables to queues

### PR DESCRIPTION
**Pull request by JeroenED**
Added bootstrap-tables to queues on the frio theme. It prettifies the output a lot.

### CODE REVIEW

File: view/theme/frio/templates/admin/queue.tpl [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div id='adminpage'>
```
Comment (MrPetovan) [inline view/theme/frio/templates/admin/queue.tpl]: Please use double quotes for HTML attributes.

File: view/theme/frio/templates/admin/queue.tpl [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	<h1>{{$title}} - {{$page}} ({{$count}})</h1>
+	
+	<p>{{$info}}</p>
+	<table class='table'>
```
Comment (MrPetovan) [inline view/theme/frio/templates/admin/queue.tpl]: Please use double quotes for HTML attributes.

File: view/theme/frio/templates/admin/workerqueue.tpl [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<div id='adminpage'>
```
Comment (MrPetovan) [inline view/theme/frio/templates/admin/workerqueue.tpl]: Please use double quotes for HTML attributes.

File: view/theme/frio/templates/admin/workerqueue.tpl [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+	<h1>{{$title}} - {{$page}} ({{$count}})</h1>
+	
+	<p>{{$info}}</p>
+	<table class='table'>
```
Comment (MrPetovan) [inline view/theme/frio/templates/admin/workerqueue.tpl]: Please use double quotes for HTML attributes.

File: view/theme/frio/templates/admin/queue.tpl (hunk 1 above)
Comment (JeroenED) [reply view/theme/frio/templates/admin/queue.tpl]: I don't want to argue with you, but you should checkout the original (view/templates/admin/queue.tpl)

I only added the class table to the the tables ;)

**Pull request merged**